### PR TITLE
Fixed donate page style and amount issue

### PR DESCRIPTION
**Pull request by SarKurd**

### CODE REVIEW

File: components/donate.vue [hunk 1]
```diff
@@ -23,7 +23,7 @@
           
           <input id="paypal" v-model="method" class="method-radio" type="radio" value="paypal">
           <label class="method-label" for="paypal">
-            <i class="pf pf-paypal-alt"/>
+            <img class="paypal-icon" src="/icons/paypal.svg">
```
Comment (M3kH) [inline components/donate.vue]: Please add `alt=""` attribute :-)

File: components/donate.vue [hunk 2]
```diff
@@ -46,6 +47,7 @@
           type="text"
           name="description"
           placeholder="Message"
+          autocomplete="off"
```
Comment (M3kH) [inline components/donate.vue]: 👍

File: components/donate.vue (hunk 1 above)
Comment (M3kH) [inline components/donate.vue]:
```suggestion
            <img class="paypal-icon" alt="" src="/icons/paypal.svg">
```

**Comment by NoerGitKat** [comment]
Looking good @SarKurd !

**Pull request merged**